yarn-project/p2p/src/services/peer-manager/peer_manager.ts

```diff
@@ -302,16 +363,16 @@ export class PeerManager {
     }
   }
 
-  private onlyNotTrustedPeers(connections: Connection[]): Connection[] {
-    return connections.filter(conn => !this.isTrustedPeer(conn.remotePeer));
+  private getNonProtectedPeers(connections: Connection[]): Connection[] {
+    return connections.filter(conn => !this.isProtectedPeer(conn.remotePeer));
   }
 
   private pruneUnhealthyPeers(connections: Connection[]): Connection[] {
     const connectedHealthyPeers: Connection[] = [];
 
     for (const peer of connections) {
-      if (this.isTrustedPeer(peer.remotePeer)) {
-        this.logger.debug(`Not pruning trusted peer ${peer.remotePeer.toString()}`);
+      if (this.isProtectedPeer(peer.remotePeer)) {
+        this.logger.debug(`Not pruning protected peer ${peer.remotePeer.toString()}`);
```

Comment:
Lets remove this log line, itll be extremely noisy, and will be logged regardless of pruning or not

Reply:
Reordered the list on line 304 to remove the check entirely